inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/activation.cpp

```diff
@@ -29,13 +29,29 @@ const std::vector<ActivationTypes> activationTypes = {
         Exp,
         Log,
         Sign,
-        Abs
+        Abs,
+        Clamp,
+        Negative,
+        Acos,
+        Asin,
+        Atan,
+        Cos,
+        Cosh,
+        Floor,
+        Sin,
+        Sinh,
+        Sqrt,
+        Tan,
+        Elu,
+        Erf,
+        HardSigmoid,
+        Selu
```

Comment:
@vladimir-paramuzov  @iefode  Does it make sense not to mix the things up and separate this into two tests (Activation and Arithmetic) according to the spec: https://gitlab-icv.inn.intel.com/inference-engine/dldt/tree/docs_master/docs/ops/activation and https://gitlab-icv.inn.intel.com/inference-engine/dldt/tree/docs_master/docs/ops/arithmetic.
Another proposal here can be to rename Activation test to Unary that will cover Eltwise operations with single input, While Eltwise would mean Binary (aka Eltwise operaitions with two inputs).
What do you think?

Reply:
Thanks @dmitry-gorokhov . Sounds good for me (First one).

Reply:
@dmitry-gorokhov @iefode I have an impression that the things have been already mixed up in the spec. 
Some examples:
 - [Selu](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/arithmetic/Selu_1.md)- for some reason the spec is located in arithmetic folder, but the description says: "Selu calculates the SELU **activation function** "
 - [Tanh](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/arithmetic/Tanh_1.md) - is also put into the arithmetic folder despite the description: "Category: **Activation function**", "Short description: Tanh element-wise **activation function**.", but [Sinh](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/arithmetic/Sinh_1.md) "performs element-wise hyperbolic sine (sinh) operation with given tensor." and relates to "Arithmetic unary operation" category.

Anyway, we won't have single test class for all activations described in https://github.com/openvinotoolkit/openvino/tree/master/docs/ops/activation, because we have separate class for Softmax.
So I'd say that the second option is preferable.

Anyway, looks like specs must be reviewed and updated. cc @lazarevevgeny 
P.S. rendering of formulas on github doesn't work. @lazarevevgeny is it a known issue?

Reply:
Issue: 34733